modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/Tabs.js

```diff
@@ -6,12 +6,14 @@
 import ClayTabs from '@clayui/tabs';
 import React, {useEffect, useRef} from 'react';
 
-const Panel = ({children}) => {
+const Panel = ({children: tabPanel}) => {
 	const ref = useRef();
 
 	useEffect(() => {
-		ref.current.appendChild(children);
-	}, [children]);
+		for (const child of tabPanel.children) {
+			ref.current.appendChild(child);
```

Comment:
We discovered during testing that this is appending one of every 2 nodes. Reason is that `appendChild()` removes the node from the source nodelist (`tabPanel.children`) when appending into `ref.current` so the `tabPanel.children` iterator is being mutated while loop traverses it.

Reply:
Hi Dani! I've updated the PR with the approach we discussed yesterday, appending the first child, so it doesn't affect the loop. Let me know what you think 😄